src/utils/writeToDataFeed.ts

```diff
@@ -11,7 +10,7 @@ export function writeToDataFeed(dataFeed: unknown) {
     state.wallet.issueOrSelectAddressByIndex(0, 0, (botWallet) => {
         const params = {
             paying_addresses: [botWallet],
-            outputs: [{ address: botWallet, amount: 0 }],
+            outputs: [{ address: botWallet, amount: 10000 }],
```

Comment:
What is the reason for this change?

Reply:
Originally it was a required amount to send a transaction, but since the new method doesn't use it, I've deleted it completely.